app/views/admin/councillor_contributions/_details.html.arb

```diff
@@ -0,0 +1,13 @@
+table do
+  tr do
+    th "Contributor Name"
+    th "Contributor Email"
+    th "Authority"
+  end
+  tr do
+    td a contributor.name, href:admin_contributor_path
+    td contributor.email
```

Comment:
A `mailto:` link could be useful to allow the admin to quickly email the contributor.